src/components/predictive-layer.vue

```diff
@@ -55,7 +89,7 @@
         </h1>
         <NextQueries
           :animation="suggestionsAnimation"
-          :max-items-to-render="5"
+          :max-items-to-render="$x.query.searchBox ? 3 : 4"
```

Comment:
The max next queries are always 3

```suggestion
          :max-items-to-render="3"
```